src/test/java/ProfileTests.java

```diff
@@ -1,31 +0,0 @@
-import org.openqa.selenium.By;
-import org.openqa.selenium.WebElement;
-import org.testng.Assert;
-import org.testng.annotations.Test;
-import pages.LoginPage;
-import pages.ProfilePage;
-
-public class ProfileTests extends BaseTest {
-
-//    @Test(enabled = true, priority = 3, description = "LoginValidEmailValidPasswordTest")
-//    public static void updateProfileNameTest () throws InterruptedException {
-//
-//        provideEmail("demo@class.com");
-//        providePassword("te$t$tudent");
-//        clickSubmit();
-//
-//        Thread.sleep(2000);
-//        clickAvatarIcon();
-//
-//        String randomName = generateRandomName();
-//
-//        provideCurrentPassword("te$t$tudent");
-//        provideProfileName(randomName);
-//        clickSaveButton();
-//
-//        Thread.sleep(2000);
-//        WebElement actualProfileName = driver.findElement(By.cssSelector("a.view-profile>span"));
-//        Assert.assertEquals(actualProfileName.getText(), randomName);
-//
-//    }
-}
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.